src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -76,6 +76,9 @@ DlgPrefRecord::DlgPrefRecord(QWidget* parent, UserSettingsPointer pConfig)
     CheckBoxRecordCueFile->setChecked(m_pConfig->getValue<bool>(
             ConfigKey(RECORDING_PREF_KEY, "CueEnabled"), kDefaultCueEnabled));
 
+    CheckBoxUseCueFileAnnotation->setChecked(m_pConfig->getValue<bool>(
+            ConfigKey(RECORDING_PREF_KEY, "CueFileAnnotationEnabled"), false));
```

Comment:
We are trying to streamline new CO and config keys to use `snake_case`
```suggestion
            ConfigKey(RECORDING_PREF_KEY, "cue_file_annotation_enabled"), false));
```